src/components/SingleDatePicker/SingleDatePicker.stories.tsx

```diff
@@ -46,3 +46,30 @@ export const WithCustomPlaceholder: Story = () => {
     />
   );
 };
+
+export const OpenDatepicker: Story = () => {
+  const [dateValue, handleChangeDate] = useState<Date>(null);
+
+  return (
+    <SingleDatePicker
+      placeholder="Custom placeholder"
+      value={dateValue}
+      isOpen
```

Comment:
I think this is ok, we expose that also for other components, but I would change it to:
* `defaultIsOpen`
* `defaultIsYearPickerOpen`
to remain consistent with other components

Reply:
I think we don't actually need to expose this, but is a nice API addition, and it also helps to have more meaningful visual regression tests, otherwise we're just testing the text input